packages/shared/src/utils/location/location.js

```diff
@@ -6,3 +6,39 @@ export const getLocation = (location_list, value, type) => {
     if (location_obj) return location_obj[type];
     return '';
 };
+
+// eu countries to support
+const eu_countries = [
```

Comment:
can we reuse this from `/packages/core/src/_common/utility.js`

Reply:
@mehrzad-fs Pointed this out [in this comment](https://github.com/binary-com/deriv-app/pull/4989#discussion_r821459583)
The same function is also declared in `cashier` which I think it's not good. IMO it's better to move them here in `shared`. I also put a todo comment to remove duplicate code later.

Reply:
@salarhfz-fs sure, sorry didnt see the message above